src/qibolab/utils.py

```diff
@@ -0,0 +1,69 @@
+"""Helper methods for loading and saving to runcards.
+
+The format of runcards in the ``qiboteam/qibolab_platforms_qrc``
+repository is assumed here. See :ref:`Using runcards <using_runcards>`
+example for more details.
+"""
+from pathlib import Path
+from typing import Tuple
+
+import yaml
+
+from qibolab.native import SingleQubitNatives, TwoQubitNatives
+from qibolab.platform import PlatformSettings, QubitMapType, QubitPairMapType
+from qibolab.qubits import Qubit, QubitPair
+
+
+def load_runcard(path: Path) -> dict:
+    """Load runcard YAML to a dictionary."""
+    return yaml.safe_load(path.read_text())
+
+
+def load_settings(runcard: dict) -> PlatformSettings:
+    """Load platform settings section from the runcard."""
+    return PlatformSettings(**runcard["settings"])
+
+
+def load_qubits(runcard: dict) -> Tuple[QubitMapType, QubitPairMapType]:
+    """Load qubits and pairs from the runcard.
```

Comment:
For the time being we could even keep as it is, to make a smoother transition, but eventually I'd prefer to make more isolated functions, that would not depend on the whole runcard and its structure, but just on the relevant section.